data-store/chapter.md

```diff
@@ -489,8 +490,8 @@ Instead of updating a node to point to a new subtree,
 we make a new node which shares the unchanged subtree.
 This is what makes this binary tree an immutable data structure.
 
-This is where we notice something strange -- we haven't made any changes to
-anything on disk yet! All we've done is manipulate our view of the on-disk data
+This is where we notice something strange: we haven't made any changes to
```

Comment:
"At this point we notice something strange" to avoid the "This… This… This…" repetition?